bin/debops-update

```diff
@@ -157,7 +160,7 @@ def update_git_repository(path, dry_run=False, remote_uri=False):
 
     if dry_run:
         subprocess.call([GIT, 'fetch'])
-        subprocess.call([GIT, 'diff', 'HEAD', 'origin', '--stat'])
+        subprocess.call([GIT, 'diff', 'HEAD', 'origin/'+branch, '--stat'])
```

Comment:
@nqb Perhaps you could reformat this change as
```python
'origin/' + branch
```
At first glance I thought that this was some weird way to pick a branch in `git`, not that `branch` was a Python variable.

Reply:
Hi @drybjed,

I agree. But since this commit, I changed my idea on how to handle this case.

With current state of this PR, if you checkout a branch with `--branch stable-1.1` option, you will need to specify `--branch stable-1.1` when you will run `debops-update --dry-run` after a first checkout. If you didn't, you will get a diff between `stable-1.1` and `master`. That's not the case for merge operation because we use `SHA`.

So my idea is to use same code for `dry_run` and `merge` operations (use `SHA`) but handle cases where users use `--branch` after a first checkout.

Let me know if that sounds good for you.

Reply:
How about getting the currently checked out branch name with
```
git rev-parse --abbrev-ref HEAD
```
and using that in the diff and dry run?